src/platforms/native/guides/minidumps/index.mdx

```diff
@@ -10,15 +10,12 @@ description: "Learn about how Sentry processes Minidump crash reports."
 Sentry can process Minidump crash reports, a memory dump used on Windows and by
 open-source libraries like [_Breakpad_](/platforms/native/guides/breakpad/) or [_Crashpad_](/platforms/native/guides/crashpad/).
 
-<Note>
 
-We strongly encourage you to use a higher-level SDK if possible, as Minidumps uploads are very limited in functionality:
+<Alert level="warning">
```

Comment:
Apply the same guideline:
- **Note:** use these for useful context, information that should be visually called out from the core content. Use sparingly.
- **Important:** use these for information that's critical to know (this version introduces breaking changes). Use infrequently.
- **Warning:** use these for items that MUST be well understood before proceeding. These should be used very rarely.

Reply:
Fixed in https://github.com/getsentry/sentry-docs/pull/3369